src/main/webapp/WEB-INF/jsp/home.jsp

```diff
@@ -0,0 +1,11 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
```

Comment:
Why did you choose ISO-8859-1? is it not a problem that your code is formatted in UTF-8 as specified in your pom.xml?

Reply:
You can test it by adding to the page some label with accents and see the page in the browser. I suppose the accents will be weird characters because they are sent to the browser as UTF-8 but the directive is indicating they must be parsed as ISO-8859-1... I recommend to change the charset everywhere to UTF-8

Reply:
I'll change that